indico/web/rh.py

```diff
@@ -327,6 +327,9 @@ def wrapper(*args, **kwargs):
 
 
 class RHProtected(RH):
+    """
+    Requires the session to have a user
+    """
```

Comment:
```suggestion
    """Base RH that requires the user to be authenticated."""
```

https://www.python.org/dev/peps/pep-0257/#one-line-docstrings